frame/benchmarking/src/utils.rs

```
@@ -118,6 +131,17 @@ impl BenchmarkResult {
 	}
 }
 
+/// Helper module to make serde serialize `Vec<u8>` as strings.
+#[cfg(feature = "std")]
+mod serde_as_str {
+	pub fn serialize<S>(value: &Vec<u8>, serializer: S) -> Result<S::Ok, S::Error>
+	where
+		S: serde::Serializer,
+	{
+		serializer.collect_str(std::str::from_utf8(value).unwrap())
```

No unwrap. You can just return the error